internal/worker/tart/tart.go

```diff
@@ -51,6 +52,15 @@ func (vm *VM) tart(
 	return stdout.String(), stderr.String(), err
 }
 
+func List(ctx context.Context, logger *zap.SugaredLogger, args ...string) ([]string, error) {
+	output, _, err := Tart(ctx, logger, "list", "-q")
```

Comment:
What about using the json output that can also return if a VM is running?

Reply:
Implemented in https://github.com/cirruslabs/orchard/pull/70/commits/d05c24f2603f6c688b8bfdf04f199e82fe9d6c74.